src/components/Creations/Creations.tsx

```diff
@@ -73,6 +74,12 @@ const Creations = (props: Props) => {
     },
     [getRaritiesQueryString, changeFilter]
   )
+  const handleSetSW = useCallback(
+    (isOnlySmart: boolean) => {
+      changeFilter('isWearableSmart', `${isOnlySmart}`)
```

Comment:
Just as an alternative:
```suggestion
      changeFilter('isWearableSmart', isOnlySmart.toString())
```